changelog.yaml

```
@@ -468,3 +468,7 @@
     added:
     - Disability benefits to COL support measures.
   date: 2022-05-26 16:16:50
+- bump: minor
```

Since no new features
```suggestion
- bump: patch
```